backend/test/modules/tournament-tree/core/domain/TournamentTree.spec.ts

```diff
@@ -1,118 +1,349 @@
 import { NumberIdGeneratorStub } from '../../../../test-support/shared/core/NumberIdGeneratorStub';
-import { TournamentTree } from '../../../../../src/modules/tournament-tree/core/domain/TournamentTree';
+import { createTournamentTree, TournamentTree } from '../../../../../src/modules/tournament-tree/core/domain/TournamentTree';
 import { generateTournamentTeamsList } from './TouramentTeamsListGenerator';
 import { FightingTeamsGroup } from '../../../../../src/modules/tournament-tree/core/domain/FightingTeamsGroup';
 import { FightingTeamsGroupId } from '../../../../../src/modules/tournament-tree/core/domain/FightingTeamsGroupId';
 
 describe('TournamentTree', () => {
-  const testTournamentId = NumberIdGeneratorStub(1000, 'testTournamentId');
-
-  it('CreateTournamentTree | Create correct 4 teams tree', async () => {
-    //Given
-    const teamEntityIdGen = NumberIdGeneratorStub(1000, 'team');
-    const matchEntityIdGen = NumberIdGeneratorStub(1000, 'match');
-    const tournamentTeamsList = generateTournamentTeamsList(teamEntityIdGen, 4);
-
-    // //When
-    const tournamentTree: TournamentTree = TournamentTree.createSingleTournamentTree({
-      tournamentId: testTournamentId.generate(),
-      tournamentTeams: tournamentTeamsList,
-      entityIdGenerator: matchEntityIdGen,
+  describe('Single Tournament Tree', () => {
+    const testTournamentId = NumberIdGeneratorStub(1000, 'testTournamentId');
+
+    it('CreateTournamentTree | Create correct 4 teams tree', async () => {
+      //Given
+      const teamEntityIdGen = NumberIdGeneratorStub(1000, 'team');
+      const matchEntityIdGen = NumberIdGeneratorStub(1000, 'match');
+      const tournamentTeamsList = generateTournamentTeamsList(teamEntityIdGen, 4);
+
+      // //When
+      const tournamentTree: TournamentTree = TournamentTree.createSingleTournamentTree({
+        tournamentId: testTournamentId.generate(),
+        tournamentTeams: tournamentTeamsList,
+        entityIdGenerator: matchEntityIdGen,
+      });
+
+      const expectedWinnerTree: FightingTeamsGroup[] = [
```

Comment:
Czy to nie jest częśc Then?

Reply:
Czy komentarze usunąć czy tylko dodać tam gdzie brakuje?

Reply:
Dodać gdzie brakuje.